datasource-reference/querying-dynamodb.md

```diff
@@ -0,0 +1,104 @@
+# DynamoDB
+
+{% hint style="warning" %}
+The following document assumes that you understand the [basics of connecting to databases on Appsmith](../core-concepts/connecting-to-databases/). If not, please go over them before reading further.
```

Comment:
@nidhi-nair this won't show up unless you add it in the SUMMARY.md
Can you also cherry pick this onto 1.2.1